specs/latest/2.0/index.html

```diff
@@ -2121,7 +2141,7 @@ <h2 class="no-toc">Table of contents</h2>
               <tr><td>TEXTURE_WRAP_T</td><td>GLenum</td></tr>
           </table>
           <p>If <em>pname</em> is not in the table above, generates an <code>INVALID_ENUM</code> error.</p>
-          <p>If an OpenGL error is generated, returns null.</p>
+          <p>If <em>sampler</em> is not a valid sampler object or an OpenGL error is generated, returns null.</p>
```

Comment:
This is confusing, because if |sampler| is not a valid sampler object, an INVALID_OPERATION will be generated, so it is not an "or" condition, but already covered by the original condition.

A better way is to add the INVALID_OPERATION error for invalid |sampler|.